Brixcer_Box/src/main.cpp

```diff
@@ -156,18 +153,18 @@ void receiveData() {
 
 void buttonClick() {
     for (int i = 0; i < KnobCount; i++) {
-        int button = knobs[i].readButtonValue();
-        if (button) {
-            sendData(i, 0, button);
```

Comment:
Why did these change, particularly to what seems to be a slightly less descriptive name?